roles/lib_openshift/library/oc_route.py

```diff
@@ -1704,18 +1690,6 @@ def run_ansible(params, check_mode=False):
                          'content': params['key_content'],
                          'value': None, }, }
 
-        if params['tls_termination'] and params['tls_termination'].lower() != 'passthrough':  # E501
-
-            for key, option in files.items():
-                if key == 'destcacert' and params['tls_termination'] != 'reencrypt':
-                    continue
-
-                option['value'] = OCRoute.get_cert_data(option['path'], option['content'])  # E501
```

Comment:
There is no need to load the data from the certificate (at least not in that module). `get_cert_data` is not called from anywhere else.

If I understood the code logic correctly, it only verifies the certificates for reencrypt method.